File: data_structures_by_c++/linked_list/singly_linked_list/creating_a_node.cpp
#include<bits/stdc++.h>
using namespace std;

#define int long long

class Node
{
    public:
        int val;
        Node* next;
        Node(int val)
        {
            this->val = val;
            this->next = NULL;
        }
};

int32_t main()
{
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    Node a(10),b(20),c(30);

    a.next = &b;
    b.next = &c;

    cout << a.val << endl;
    cout << a.next->val << endl;
    cout << a.next->next->val << endl;
}

File: data_structures_by_python/linked_list/singly_linked_list/creating_a_node.py
class Node:
    def __init__(self, val):
        self.val = val
        self.next = None


a = Node(10)
b = Node(20)
c = Node(30)
a.next = b
b.next = c

print(a.val)
print(a.next.val)
print(a.next.next.val)
